modules/reference/pages/home/home-my-preferences.adoc

```diff
@@ -14,8 +14,11 @@ The menu:Home[My Preferences] section allows you to configure {productname} {web
 | {productname} List Page Size | Maximum number of items that can appear in a list on a single page. | 25 entries
 | "Overview" Start Page | Select the information panes to display on the menu:Home[Overview] page. | All checked
 | Time Zone             | Set your local timezone. | System timezone
+| Style Theme           | Set the style theme to use in the {webui}. | Default `susemanager-light` (*)
 | CSV Files             | Select whether to use comma or semi-colon delimiters when producing downloadable CSV files. | Comma
 |===
 
 
+(*) The style theme default value can be changed in the [path]``rhn.conf`` by setting the ``web.theme_default`` parameter to one of the values in the ``web.themes`` parameter list. The list itself can be customized by adding or removing any value: this will be reflected in the options set of the style theme selection. By default the theme list contains: ``susemanager-light``, ``susemanager-dark`` and ``uyuni``. Any item of this list is supposed to be bounded to the name of each [path]``/srv/www/htdocs/css/<style-file-name>.css``
```

Comment:
This needs to be moved to modules/installation/pages/webui-setup.adoc and an xref included in the table. If you want this merged today, I say we take L22 out for now and create an issue to get it added to the Install Guide after we've packaged, though.

Reply:
I removed it for now, creating the issue https://github.com/uyuni-project/uyuni-docs/issues/614